tests/unit_test/api/api_processor_test.py

```
@@ -1191,14 +1191,14 @@ def test_send_confirmation_link_with_valid_id(self, monkeypatch):
 
-    def test_send_confirmation_link_with_confirmed_id(self, monkeypatch):
+    @pytest.mark.asyncio
+    async def test_send_confirmation_link_with_confirmed_id(self, monkeypatch):
         Utility.email_conf["email"]["enable"] = True
+        UserEmailConfirmation(email='integ1@gmail.com').save()
         monkeypatch.setattr(MailUtility, 'trigger_smtp', self.mock_smtp)
-        loop = asyncio.new_event_loop()
-        with pytest.raises(Exception):
-            loop.run_until_complete(AccountProcessor.send_confirmation_link('integ1@gmail.com'))
+        with pytest.raises(AppException):
+            res =await AccountProcessor.send_confirmation_link('integ1@gmail.com')
```

Ruff (0.8.2)

Local variable `res` is assigned to but never used

Remove assignment to unused variable `res`

(F841)